dev/io.openliberty.jakartaee10.internal_fat/fat/src/io/openliberty/jakartaee10/internal/tests/EE10FeatureCompatibilityTest.java

```diff
@@ -474,8 +430,8 @@ public void transportSecurityUsed() {
         ee10FeaturesThatEnableSsl.add("connectorsInboundSecurity-2.0");
         ee10FeaturesThatEnableSsl.add("messagingSecurity-3.0");
 
-        for (String feature : features) {
-            if (nonEE10JavaEEFeatures.contains(feature) || nonEE10MicroProfileFeatures.contains(feature) || incompatibleValueAddFeatures.contains(feature)) {
+        for (String feature : allFeatures) {
+            if (!compatibleFeatures.contains(feature)) {
                 continue;
             }
```

Comment:
```suggestion
        for (String feature : compatibleFeatures) {
```

Didn't have a compatibility list previously so had to iterate over all features and ignore incompatible features.  But with having a compatibility list can just iterate over those.

Reply:
Ok, that does make way more sense

Reply:
The change needs to be applied to EE9 test as well.  I didn't state it earlier, but any comments I made in the EE10 test apply to the EE9 test as well.

Reply:
Done